### PR TITLE
Fixed bug and added more tests

### PR DESCRIPTION
**Pull request by rileymckenna**
## Description

Fixed bug in get Peering location 
Added more tests that would have caught this bug 
Added more tests to direct connection

## Checklist

- [ ] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by msJinLei** [comment]
@rileymckenna could you provide the issue reference of the fix & please fix the test error.

**Comment by rileymckenna** [comment]
https://github.com/Azure/azure-powershell/issues/10297

**Comment by rileymckenna** [comment]
@msJinLei what needs to be revised?

File: src/Peering/Peering/Peering/GetAzurePeeringLocationCommand.cs [hunk 1]
```diff
@@ -73,8 +73,7 @@ public class GetAzurePeeringLocationCommand : PeeringBaseCmdlet
         /// Gets or sets the Direct Peering Type
         /// </summary>
         [Parameter(
```
[the remark this answers is not in the document]
Comment (rileymckenna) [reply src/Peering/Peering/Peering/GetAzurePeeringLocationCommand.cs]: [According to the documentation](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-attribute-help.md#cmdletparameterbreakingchangeattribute), this is not a case where i have to add a breaking change. The mandatory field is supposed to be false.

**Comment by rileymckenna** [comment]
[According to the documentation](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-attribute-help.md#cmdletparameterbreakingchangeattribute), this is not a case where i have to add a breaking change. The mandatory field is supposed to be false.  @msJinLei

**Comment by msJinLei** [comment]
> [According to the documentation](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-attribute-help.md#cmdletparameterbreakingchangeattribute), this is not a case where i have to add a breaking change. The mandatory field is supposed to be false. @msJinLei
You remove positioning attribute of a parameter.

**Comment by wyunchi-ms** [comment]
Hi @rileymckenna , this pr last for a long time. Do you still need this pr? If yes, could you please take a look at this issue? We may close this issue in a few days, if it's no longger needed.

**Comment by msJinLei** [comment]
> Hi @rileymckenna , this pr last for a long time. Do you still need this pr? If yes, could you please take a look at this issue? We may close this issue in a few days, if it's no longer needed.

@rileymckenna You should keep the positioning attribute and add a breaking change notification in the current version and remove positioning attribute in the next breaking change release.
For simplicity, you can just keep that attribute so that it is no breaking change.

**Comment by rileymckenna** [comment]
@msJinLei i apologize i was out of the office for the beginning of the week.

**Pull request merged**